tutorials/2_sprite_animations_gestures/README.md

```diff
@@ -0,0 +1,241 @@
+# Basic: Sprites, Animations and Gestures
+
+This tutorial will introduce you to:
+
+ - `Sprite`: Sprites are how we draw images, or portions of an image in Flame.
+ - `SpriteAnimation`: SpriteAnimations are animations composed from sprites, where each sprite represents a frame.
+ - Gesture input: Input is a crucial part of game development, on this tutorial you will learn how to use one of the most basic of them, the tap detector.
+
+All the code of this tutorial code can be found [here](./code) and is based on the Flame `1.0.0-rc8` version.
+
+By the end of this tutorial you will have built a simple game which renders a button which when pressed, makes a small vampire robot run, that will look like this:
```

Comment:
```suggestion
By the end of this tutorial you will have built a simple game that renders a button which, when pressed, makes a small vampire robot run. It will look like this:
```
You overuse commas @luanpotter! :smile: 
https://courses.lumenlearning.com/olemiss-writing100/chapter/comma-overuse/

Reply:
Completely changed my mind after reading through half of it, the link is for you @erickzanardo :smile:

Reply:
Why for me? I overuse commas too? hahahaha

Reply:
@spydon yeah I was gonna say, I am trying to reduce it! haha

Reply:
I don't think overusing commas is necessarily a problem, it's a style choice. But a lot of cases you use commas are grammatically wrong @erickzanardo (I will need some extra time to explain properly, hopefully my example fixes help for now)

Reply:
@luanpotter sure, it's a style choice. But for texts like this one, if there are, say, three commas or more, I would take a very careful look if the sentence isn't better suited to be split up into several parts, or formed in another way. Since the flow of reading might otherwise be disrupted. (Not sure if the example was obvious :sweat_smile:)

Reply:
I do one even better: your whole comment in a single phrase! :D 

> Sure, it's a style choice; but for texts like this one, if there are, say, three commas or more, I would take a very careful look if the sentence isn't better suited to be split up into several parts, or formed in another way -- since the flow of reading might otherwise be disrupted.

What I think is bad and disrupts the flow, for me, are misplaced commas. I don't see any problem with longer phrases but I agree they deserve closer inspection just in case there are better formulations (or errors lurking).